gnome-initial-setup/pages/keyboard/cc-input-chooser.c

```diff
@@ -169,7 +169,14 @@ preview_cb (GtkLabel       *label,
 	InputWidget *widget;
 	const gchar *layout;
 	const gchar *variant;
-	gchar *commandline;
+	gchar *lc_msg = setlocale (LC_MESSAGES, NULL);
+	gchar *command[] = {"/usr/bin/gkbd-keyboard-display", "-l", NULL, NULL};
+	gchar **env;
+	GError *error = NULL;
+	GPid pid;
+
+	env = g_get_environ ();
+	env = g_environ_setenv (env, "LC_MESSAGES", lc_msg, TRUE);
```

Comment:
This was set to `priv->new_locale_id` before -- any reason now it's set to the return value of `setlocale()`?